### PR TITLE
go.mod: bump wireguard/windows, which moves to using net/netip

### PR DESCRIPTION
**Pull request by bradfitz**
Updates #5162

### CODE REVIEW

**Comment by bradfitz** [comment]
@maisem, I've only tested that this compiles. I haven't run it on Windows. I might've screwed up something.

**Comment by bradfitz** [comment]
Get a chance to try this on Windows?

**Comment by bradfitz** [comment]
Ugh, this was never submitted.

Oh well. After 1.30 I guess.

File: wgengine/router/ifconfig_windows.go [hunk 1]
```diff
@@ -408,15 +400,13 @@ func configureInterface(cfg *Config, tun *tun.NativeTun) (retErr error) {
 		return fmt.Errorf("syncAddresses: %w", err)
 	}
 
-	sort.Slice(routes, func(i, j int) bool { return routeLess(&routes[i], &routes[j]) })
+	sort.Slice(routes, func(i, j int) bool { return routeDataCompare(&routes[i], &routes[j]) == -1 })
```
Comment (maisem) [inline wgengine/router/ifconfig_windows.go:403]: you can also do `slices.Sort(routes, routeDataLess)`

**Review by maisem — APPROVED** [review]

File: wgengine/router/ifconfig_windows.go (hunk 1 above)
Comment (bradfitz) [reply wgengine/router/ifconfig_windows.go]: done + tested

**Pull request merged**